frontend/.gitignore

```
@@ -4,3 +4,5 @@
 /.react-router/
 /build/
 .vercel
+
+.env
```

```suggestion
*.env
```

ignore all files ending by .env, e.g. if someone has `local.env` or `prod.env` those should also be ignored.